build-system/pr-check/remote-tests.js

```diff
@@ -47,10 +47,7 @@ async function main() {
 
     await startSauceConnect(FILENAME);
     timedExecOrDie('gulp unit --nobuild --saucelabs');
-    timedExecOrDie(
-      'gulp integration --nobuild --compiled --saucelabs --stable'
-    );
-    timedExecOrDie('gulp integration --nobuild --compiled --saucelabs --beta');
+    timedExecOrDie('gulp integration --nobuild --compiled --saucelabs');
```

Comment:
Combining beta and stable into a single invocation will break two things:

1) Test status reporting relies on stable / beta flags to report results to different statuses.
2) Stable failures should fail the CI job, while beta failures shouldn't.

You'll need to make some changes to the code that does the batching. Looking up past PRs that modify batching should shed light on what changes are needed.

Reply:
Merging these commands was the easiest way to test out the batching, but not actually necessary. I'm reverting the changes here and will update the tasks themselves to batch, which should piggy-back off of the existing logic for reporting failures and updating the status.

Reply:
Done. Reporting is fixed now :+1: